x/bep3/spec/01_concepts.md

```diff
@@ -0,0 +1,36 @@
+# Concepts
+
+ BEP3 is a protocol for securely transferring assets between Kava and chains that implement the BEP3 protocol via Atomic Swaps such as Binance Chain. Tranactions are witnessed and relayed between the two blockchains by Binance's BEP3 deputy process. The deputy maintains an address on both chains and is responsible for delivering tokens upon the successful completion of an Atomic Swap. Learn more about the BEP3 deputy process [here](https://github.com/binance-chain/bep3-deputy).
```

Comment:
I would link to original Binance Chain BEP3 doc: https://github.com/binance-chain/BEPs/blob/master/BEP3.md

Reply:
https://github.com/Kava-Labs/kava/pull/450/commits/2516b8d15cef7d1fa013aaef9ddb75171e7ad413